packages/core/src/components/private/BaseInputRange/styles.ts

```diff
@@ -0,0 +1,108 @@
+import { StyleSheet } from '../../../hooks/useStyles';
+import toRGBA from '../../../utils/toRGBA';
+
+export const HANDLE_SIZE = 14;
+export const HALF_HANDLE_SIZE = HANDLE_SIZE / 2;
+export const TRACK_HEIGHT = 4;
+export const MARK_SIZE = TRACK_HEIGHT;
+
+export const stylesheetInputRange: StyleSheet = ({ unit, ui, color, font, pattern }) => ({
+  container: {
+    position: 'relative',
+    height: 3 * unit,
+  },
+
+  container_disabled: {
+    opacity: 0.4,
+  },
+
+  input: {
+    transform: 'translateY(-2px)',
+    '-webkit-appearance': 'none',
+    width: '100%',
+    height: TRACK_HEIGHT,
+    margin: 0,
+    background: color.core.neutral[2],
+    outline: 'none',
+    overflow: 'visible',
+    cursor: 'pointer',
+
+    '@selectors': {
+      ':hover::-webkit-slider-thumb, :active::-webkit-slider-thumb, :focus::-webkit-slider-thumb': {
+        boxShadow: `0 0 0 2px ${color.accent.borderActive}, 0 0 0 5px ${toRGBA(
+          color.accent.borderActive,
+          20,
+        )}`,
+      },
+
+      '::-webkit-slider-thumb': {
+        '-webkit-appearance': 'none',
+        zIndex: 1,
+        transition: `opacity ${ui.transitionTime}`,
+        width: HANDLE_SIZE,
+        height: HANDLE_SIZE,
+        border: 'none',
+        borderRadius: HANDLE_SIZE,
+        background: color.base,
+        boxShadow: `0 0 0 2px ${color.accent.borderActive}`,
+      },
+    },
+  },
+
+  input_disabled: {
+    '@selectors': {
+      ':hover::-webkit-slider-thumb': { boxShadow: `0 0 0 2px ${color.accent.borderActive}` },
+    },
+  },
+
+  annotation: {
+    position: 'absolute',
+    top: 1.25 * unit + 1, // align in track center
```

Comment:
will update to `TOTAL_HEIGHT - 1` so it's in sync with the height of the container